src/hooks/use-action-dispatcher.js

```diff
@@ -0,0 +1,17 @@
+function identity(sender) {
+  return sender;
+}
+
+export default function useActionDispatcher(
```

Comment:
This function could probably be inlined into `use-deck-state` or removed altogether (in favor of directly using `React.useCallback`).